Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+| Authorization                       | ✓                                             | ✓                                                                   | ✓                            |
+| Automation                          | ✓ Webhooks                                    | ✓ Webhooks, building images                                         | ✗                            |
+| Vulnerability scanning              | ✓ Trivy, Clair                                | ✓ Clair                                                             | ✗                            |
+| Content Trust and Validation        | ✓ Cosign                                      | ✓ Cosign                                                            | ✗                            |
+| Multi-tenancy                       | ✓ (not on the storage level)                  | ✓ (not on the storage level)                                        | ✓ (not on the storage level) |
+| Backup and restore                  | ✓                                             | ✓                                                                   | ✗                            |
+| Monitoring                          | ✓ Prometheus metrics                          | ✓ Prometheus metrics                                                | ✓ Prometheus metrics         |
+| HA mode                             | ✓                                             | ✓                                                                   | ✗                            |
+| Registry replication                | ✓                                             | ✓                                                                   | ✓                            |
+| Proxy cache                         | ✓                                             | ✓ Feature is in the technology preview stage (non production ready) | ✗                            |
+| Quota management                    | ✓ Based on storage consumption or image count | ✓ Based on storage consumption                                      | ✗                            |
+| Garbage collection                  | ✓                                             | ✓                                                                   | ✗                            |
+| Retention policy                    | ✓ Multiple tag retention rules                | ✓ Only tag expiration rules                                         | ✗                            |
+| Additional supported artifacts      | ✗ (only OCI artifacts)                        | ✗ (only OCI artifacts)                                              | ✓ Maven, YUM                 |
+| Integration possibilities           | ✓ Dragonfly (P2P), Kraken (P2P)               | ✗                                                                   | ✓ Harbor, Nydus, eStargz     |
+
```

Comment:
Edge capabilities or multi regional distribution. Might be worth adding

Reply:
I found edge capabilities mentioned only in the Harbor [roadmap](https://github.com/goharbor/harbor/blob/main/ROADMAP.md#current-roadmap), can you please point us to some relevant documentation?

The table mentions Registry replication, e.g. Harbor [replication](https://goharbor.io/docs/2.7.0/administration/configuring-replication/) but we didn't find any other relevant information. Do you mean something else? I also found Harbor's issue with real-world scenarios, where the replication is also [mentioned](https://github.com/goharbor/community/issues/115#issuecomment-567822398) in the multi-DC deployment.

Reply:
It was more, a question if that is a requirement to SCS? It's on Harbors Roadmap.